cmd/cleanup.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func CleanupCmd(fs afero.Fs) *cobra.Command {
+	cleanupCmd := &cobra.Command{
+		Use:  "cleanup",
+		Long: "Delete all fragments",
+		Args: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path, err := cmd.Flags().GetString("path")
```

Comment:
Please use [`fragment_location`](https://github.com/elastic/elastic-agent-changelog-tool/blob/f2e8d6613fc17aeba181e44f3e2e024b59a36d9b/internal/settings/settings.go#L57) instead. You can get it with `viper.GetString("fragment_location")`

More docs: https://github.com/elastic/elastic-agent-changelog-tool/blob/5060e4d44fd8611e7f977ff2aa4dc78a43082d47/docs/configuration.md